base/cvd/cuttlefish/host/commands/cvd/server_command/generic.cpp

```diff
@@ -174,7 +174,10 @@ Result<cvd::Response> CvdGenericCommandHandler::Handle(
   if (is_non_help_cvd && ui_response_type == UiResponseType::kNoTTY) {
     return CF_EXPECT(NoTTYResponse(request));
   }
-
+  if (group_opt) {
+    // Update `ANDROID_HOST_OUT` with selected group's host artifacts path.
+    invocation_info.envs[kAndroidHostOut] = group_opt->HostArtifactsPath();
```

Comment:
nit: `ExtractInfo` is a better place for this, around line 398.

Reply:
good point, done.